README.md

```diff
@@ -312,7 +312,9 @@ GLOBAL OPTIONS:
    --debug                                debug output [$DEBUG, $PLUGIN_DEBUG]
    --debug-templates                      debug template output [$DEBUG_TEMPLATES, $PLUGIN_DEBUG_TEMPLATES]
    --dryrun                               if true, kd will exit prior to deployment [$DRY_RUN]
+   --delete                               instead of applying the resources we are deleting them
```

Comment:
this should be really a separate commit :)

Reply:
yep - I guess :)